chart/mits/templates/job.yaml

```
@@ -33,41 +32,24 @@ spec:
       - name: mits
         image: {{ default (printf "splatform/mits:%s" .Chart.Version) .Values.imageOverride | quote }}
         imagePullPolicy: IfNotPresent
-        command: [bash]
-        args: [{{ $runMountPath | quote }}]
+        args:
+        - ginkgo
+        {{- if .Values.ginkgo.verbose }}
+        - -v
+        {{- end }}
+        - -nodes={{ .Values.ginkgo.nodes }}
+        - -slowSpecThreshold={{ .Values.ginkgo.slow_spec_threshold }}
+        - -flakeAttempts={{ .Values.ginkgo.flake_attempts }}
+        - ./mits/mits.test
         env:
-        - name: CONFIG_TESTS
-          value: {{ printf "%s/tests.yaml" $configMountPath | quote }}
-        - name: CF_USERNAME
-          valueFrom:
-            secretKeyRef:
-              name: {{ printf "%s-cf-cred" .Release.Name | quote }}
-              key: username
-        - name: CF_PASSWORD
-          valueFrom:
-            secretKeyRef:
-              name: {{ printf "%s-cf-cred" .Release.Name | quote }}
-              key: password
-        - name: CF_ORG
-          value: {{ printf "mits-%s" (randAlphaNum 8) | quote }}
-        - name: CF_SPACE
-          value: {{ printf "mits-%s" (randAlphaNum 8) | quote }}
-        - name: CF_SERVICE_BROKER_NAME
-          value: {{ printf "minibroker-%s" (randAlphaNum 8) | quote }}
+        - name: CONFIG_PATH
+          value: {{ printf "%s/config.yaml" $configMountPath | quote }}
```

There's no reason to not hard code this, given that it _is_ hard coded on line 17.
```suggestion
          value: /mits/config/config.yaml
```

---

It's a reusable constant being declared on line 17. I'm fine with hardcoding it twice... not a big deal...

---

Done with b31a730.